rasa/nlu/training_data/formats/markdown.py

```diff
@@ -65,19 +63,6 @@ def reads(self, s: Text, **kwargs: Any) -> "TrainingData":
                 self._parse_item(line)
                 self._load_files(line)
 
-        if self._deprecated_synonym_format_was_used:
-            raise_warning(
-                "You are using the deprecated training data format to declare synonyms."
-                " Please use the following format: \n"
-                '[<entity-text>]{"entity": "<entity-type>", "value": '
-                '"<entity-synonym>"}.'
-                "\nYou can use the following command to update your training data file:"
-                "\nsed -i -E 's/\\[([^)]+)\\]\\(([^)]+):([^)]+)\\)/[\\1]{"
-                '"entity": "\\2", "value": "\\3"}/g\' nlu.md',
-                category=FutureWarning,
-                docs=DOCS_URL_TRAINING_DATA_NLU,
-            )
-
```

Comment:
Have we tested how this one breaks if you provide the wrong format? Just wondering because this is one i'm almost certain people will run into (having seen the deprecation warning before but not removing it)

Also I don't see where we removed the implementation of the functionality still working during deprecation phase

Reply:
I have no idea yet - but it looks like we dropped support for this for some time now

Reply:
oh weird 👀